### PR TITLE
added alert component

### PR DESCRIPTION
**Pull request by MiryalaAkshitha**

### CODE REVIEW

**Review by rohit8686 — APPROVED** [review]

File: components/alert/alert.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+          <div class="alert success-alert"><i class="far fa-check-circle icon"></i>   This   is    a success alert!</div>
+          <div class="alert warning-alert"><i class="fas fa-exclamation-triangle icon"></i></i>This a warning alert!</div>
+          <div class="alert info-alert"><i class="fas fa-info-circle icon"></i>This is a info alert!</div>
+          <div class="alert error-alert"><i class="far fa-times-circle icon"></i> This is a error alert! </div>
+        
+        <div class="alert outlined-success-alert outlined"><i class="far fa-check-circle icon"></i>   This   is    a success alert!</div>
+          <div class="alert outlined-warning-alert outlined"><i class="fas fa-exclamation-triangle icon"></i></i>This a warning alert!</div>
+          <div class="alert outlined-info-alert outlined"><i class="fas fa-info-circle icon"></i>This is a info alert!</div>
+          <div class="alert outlined-error-alert outlined"><i class="far fa-times-circle icon"></i> This is a error alert! </div>
+        </div>
```
Comment (romabulani) [inline components/alert/alert.html:22]: You can use prettier for good indentation

File: components/alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    place-content:center;
+    height:auto;
+    font-size:18px; 
+  }
+  .alert{
+    display:flex;
+    width:400px;
+    justify-content:center;
+    align-items:center;
+  margin-top:2rem;
```
Comment (romabulani) [inline components/alert/alert.css]: Here also indentation

File: components/alert/alert.css [hunk 3]
```diff
@@ -0,0 +1,52 @@
+.alert-container{
+    display:grid;
+    place-content:center;
+    height:auto;
+    font-size:18px; 
+  }
+  .alert{
+    display:flex;
+    width:400px;
```
Comment (romabulani) [inline components/alert/alert.css]: You should rems instead of pxs

**Review by romabulani — APPROVED** [review]
Rest all looks good to merge

File: components/alert/alert.css (hunk 2 above)
Comment (MiryalaAkshitha) [reply components/alert/alert.css]: sure roma! thankyou

**Review by anuj-kumary — APPROVED** [review]
Looks Good!!

**Pull request merged**